sound/soc/sof/sof-priv.h

```diff
@@ -56,6 +56,12 @@
 #define SOF_FORMATS (SNDRV_PCM_FMTBIT_S16_LE | SNDRV_PCM_FMTBIT_S24_LE | \
 	SNDRV_PCM_FMTBIT_S32_LE | SNDRV_PCM_FMTBIT_FLOAT)
 
+#define DMA_CHAN_INVALID	0xFFFFFFFF
+
+#define ENABLE_DEBUGFS_CACHEBUF \
+	(IS_ENABLED(CONFIG_SND_SOC_SOF_DEBUG_ENABLE_DEBUGFS_CACHE) || \
+	 IS_ENABLED(CONFIG_SND_SOC_SOF_DEBUG_IPC_FLOOD_TEST))
```

Comment:
This seems to only be used once, if that's indeed the case, maybe you don't need to define it at all.

Reply:
this one I like as it, otherwise it'd make the structure less readable.